pkg/local-gadget-manager/local-gadget-manager.go

```diff
@@ -344,13 +345,15 @@ func NewManager() (*LocalGadgetManager, error) {
 	containerEventFuncs = append(containerEventFuncs, l.containersMap.ContainersMapUpdater())
 	containerEventFuncs = append(containerEventFuncs, l.tracerCollection.TracerMapsUpdater())
 
-	err = l.ContainerCollection.ContainerCollectionInitialize(
+	opts := []containercollection.ContainerCollectionOption{
 		containercollection.WithPubSub(containerEventFuncs...),
 		containercollection.WithCgroupEnrichment(),
 		containercollection.WithLinuxNamespaceEnrichment(),
-		containercollection.WithDockerEnrichment(),
 		containercollection.WithRuncFanotify(),
-	)
+	}
+	opts = append(opts, containercollection.WithContainerRuntimesEnrichment(runtimes)...)
```

Comment:
It wasn't possible because it returns a slice so it is needed to use `append` + `...`. Or maybe I didn't find the way 😅.

Anyway, after your suggestion [here](https://github.com/kinvolk/inspektor-gadget/pull/606#discussion_r861804410), I moved it into the initialization of `opts`. Thanks!